### PR TITLE
Namespace

### PR DESCRIPTION
**Pull request by oozoofrog**
insert `sl` namespace for config, sublayout, anchors, eraseToAnyLayout, identifying of UIView extension.
and adding deprecated warning for old apis.

before
```
self
  .config { ... }
  .sublayout {
    ...
  }
  .anchors {
    ...
  }
```

after
```
self
  .sl.config { ... }
  .sl.sublayout {
    ...
  }
  .anchors {
    ...
  }
```

Others
- fix ViewPrinter
- fix testcases for changing namespace

### CODE REVIEW

File: Sources/SwiftLayout/Extension/UIVIew+Layout.swift [hunk 1]
```diff
@@ -20,7 +20,7 @@ public extension _UIViewExtension where Self: UIView, Self: LayoutElement {
     /// - Parameter build: A ``LayoutBuilder`` that  create sublayouts of this view.
     /// - Returns: An ``ViewLayout`` that wraps this view and contains sublayouts .
     ///
-    @available(*, deprecated, message: "Use the sublayout(_:) method instead.")
+    @available(*, deprecated, message: "Use the sl.sublayout(_:) method instead.")
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Extension/UIVIew+Layout.swift:23]: 아 이거 리버트 했던 사실을 잊어버렸네요.

File: Sources/SwiftLayout/Layoutable/Layoutable.swift [hunk 2]
```diff
@@ -5,7 +5,7 @@
 //  Created by oozoofrog on 2022/02/06.
 //
 
-public protocol Layoutable: AnyObject, LayoutElement {
+public protocol Layoutable: AnyObject, LayoutBase {
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Layoutable/Layoutable.swift]: I am very much for this.

File: Sources/SwiftLayout/Extension/Wrpper/LayoutMethodWrapper.swift [hunk 3]
```diff
@@ -1,31 +1,17 @@
 //
-//  LayoutableMethodWrapper.swift
+//  LayoutMethodWrapper.swift
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Extension/Wrpper/LayoutMethodWrapper.swift]: i am very much for this too.

**Review by gmlwhdtjd — APPROVED** [review]
The why and how of namespacing is very good. However, it is unfortunate that when writing layouts, we have to add `sl` every time we access layout methods like `anchor(_:)` and `sublayout(_:)` in a UIView. But this is an essential part of namespacing, and there doesn't seem to be any way to avoid it. 

Thanks for your efforts.

**Comment by oozoofrog** [comment]
@gmlwhdtjd 
non inline history made branch merging impossible, so i cleaned it up with a force push. pr has been refreshed.

**Review by gmlwhdtjd — APPROVED** [review]

**Pull request merged**